index.js

```diff
@@ -12,7 +12,7 @@ const createHomeComponent = require('./templates/home-component/create-home-comp
 const createPackageJson = require('./templates/package-json/create-package-json');
 const filterFiles = require('./utils/filter-files');
 
-module.exports = async function(projectPath) {
+module.exports = async function(projectPath, callback) {
```

Comment:
Consider using a promise instead of callback here. 
One extra parameter for some CPS << one level of indention